src/polygonFeature.js

```diff
@@ -8,6 +8,27 @@ var feature = require('./feature');
  *
  * @class geo.polygonFeature
  * @extends geo.feature
+ * @param {Object} arg Options object
+ * @param {Object|Function} [arg.position] Position of the data.  Default is
+ *   (data).  The position is an Object which specifies the location of the
+ *   data in geo-spatial context.
```

Comment:
We describe position elsewhere with this same text elsewhere, too.  I'll
change it.

On Sat, Jun 25, 2016 at 12:20 AM, Aashish Chaudhary <
notifications@github.com> wrote:

> In src/polygonFeature.js
> <https://github.com/OpenGeoscience/geojs/pull/594#discussion_r68483789>:
>
> > @@ -8,6 +8,27 @@ var feature = require('./feature');
> >   *
> >   * @class geo.polygonFeature
> >   * @extends geo.feature
> > + * @param {Object} arg Options object
> > + * @param {Object|Function} [arg.position] Position of the data.  Default is
> > + *   (data).  The position is an Object which specifies the location of the
> > + *   data in geo-spatial context.
>
> minor suggestion: geo-spatial context is not a must.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGeoscience/geojs/pull/594/files/67fa92de4928f8d849ff0abbbb2b8dabbe14eb1e#r68483789>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIX_Rx0mnlk2llX-t1ZSXeDCfbAgp4Bxks5qPKyOgaJpZM4I3hEE>
> .
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439